### PR TITLE
add recipe for LexikCronFileGeneratorBundle 

### PR DESCRIPTION
**Pull request by spike31**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by spike31** [comment]
@Nyholm 

Hi, 
Are the PRs not supposed to be merged automatically?

**Comment by spike31** [comment]
@weaverryan 

Hi,
Maybe is related with your issue on flex recipe, but how recipes contribs are handled?
I saw some PR approuved but never merged, what is the process for PR on recipes-contrib?

The recipes-contribs are something interesting, but in addition to maintaining the bundles, it is also necessary to maintain the recipes-contribs and wait not sure why or who to ask ... some Prs are merged, others remain on the roadside... it's a "mystère"

Thanks if someone read this :D

File: lexik/cron-file-generator-bundle/1.0/config/packages/lexik_cron_file_generator.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# see https://github.com/lexik/LexikCronFileGeneratorBundle/blob/master/Resources/doc/index.md
+lexik_cron_file_generator:
+    env_available:                 # declare your env availables
+        - staging                    # example: staging, prod
+    user:
+        staging: 'project_staging'     # user for execute the cron
+    php_version: 'php7.2'
+    absolute_path:
+        staging: 'project_dir/bin/console'
```
Comment (Nyholm) [inline lexik/cron-file-generator-bundle/1.0/config/packages/lexik_cron_file_generator.yaml]: Do you mean `'%kernel.project_dir%/bin/console'`?

Comment (spike31) [reply lexik/cron-file-generator-bundle/1.0/config/packages/lexik_cron_file_generator.yaml]: No it's juste a path on the server, not with symfony

**Comment by spike31** [comment]
@Nyholm No problem :)

**Pull request merged**